spec/spec_helper.rb

```diff
@@ -1,3 +1,10 @@
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start do
+    track_files '{app,lib}/**/*.rb'
```

Comment:
You only want the `track_files` for Rails apps.  Just use `SimpleCov.start` here.